testcafe/e2e/changeLanguage.ts

```diff
@@ -0,0 +1,27 @@
+import { screen } from '@testing-library/testcafe';
+
+import { header } from '../selectors/header';
+import { getPathname } from '../utils/clientUtils';
+import { getEnvUrl } from '../utils/settings';
+
+fixture('Landing page').page(getEnvUrl('/fi/home'));
+
+test('Changing language on landing page', async (t) => {
+  await t
+    .expect(screen.getAllByRole('link', { name: /etsi tekemistä/i }).exists)
```

Comment:
are there many links with text "etsi tekemistä"?

Reply:
if there is one, maybe getByRole is better. If there are many, could the the amount of test be tested?

Reply:
Both footer and header have those links. I started testing exact amount of elements e.g. 
```
await t
    .expect(screen.getAllByRole('link', { name: /etsi tekemistä/i }).count)
    .eql(2)
    .expect(screen.getAllByRole('link', { name: /suosittelemme/i }).count)
    .eql(2);
```